app/src/main/java/org/fossasia/openevent/general/event/topic/SimilarEventsFragment.kt

```diff
@@ -159,8 +159,8 @@ class SimilarEventsFragment : Fragment() {
 
         handleVisibility(similarEvents)
         Timber.d("Fetched Similar events of size %s", similarEvents.size)
-        if (similarEventsListAdapter.currentList.size != similarEvents.size) similarEvents.shuffle()
-        similarEventsListAdapter.submitList(similarEvents)
+//        if (similarEventsListAdapter.currentList.size != similarEvents.size) similarEvents.shuffle()
+//        similarEventsListAdapter.submitList(similarEvents)
```

Comment:
Why?

Reply:
Will implement pagination here as well it uses same adapters event view holder

Reply:
@iamareebjamal can you tell why doFinally Block is not working here because of that shimmer layout do not stop everything else just works fine. is it because pagination requires continuous API calls to different pages?

Reply:
@iamareebjamal should i implement pagination here as well because this method won't no longer work as the adapter expects as pagedlist and we are passing a normal list here

Reply:
And there would be a heavy network usage if I implement pagination here as well I suggest we should create a seperate adapter for this and use this as a normal list

Reply:
@iamareebjamal updated